src/rules/use-components.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-present Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const BPK_SUBSTITUES = {
+  Text: 'BpkText',
+  TouchableNativeFeedback: 'BpkTouchableNativeFeedback',
```

Comment:
We also have 

+ `Button` instead of `BpkButton`
+ `ActivityIndicator` instead of `BpkSpinner`
+ `Alert` instead of `BpkAlert`

Reply:
I did only those two because that was what the card asked for. But I guess we might as well add those you mentioned too.

Reply:
Sorry I didn't know that. Might be the intention of the card to start with a small set. We can always add new cases later

Reply:
@matthewdavidson what do you think?

Reply:
If it's easy, then add em all :)

Reply:
There is also `BpkTouchableOverlay` instread of `TouchableHighlight`

Reply:
Also `BpkImage` instead of `Image` now.